universal-login-sdk/lib/sdk.ts

```diff
@@ -212,8 +212,9 @@ class UniversalLoginSDK {
     throw `Unknown event type: ${eventType}`;
   }
 
-  subscribeAuthorisations(walletContractAddress: string, callback: Function): () => void {
-    return this.relayerObserver.subscribe(walletContractAddress, callback);
+  subscribeAuthorisations(getAuthorisationRequest: GetAuthorisationRequest, callback: Function) {
```

Comment:
Here shouldn't it be more like:
`subscribeAuthorisations(walletContractAddress: string, privateKey: string, callback: Function)`,
so that user don't need to sign on his own?